app/views/grape_swagger_rails/application/index.html.erb

```diff
@@ -263,6 +263,9 @@
             <li>
               <a href="http://www.gov.uk/help/cookies">Cookies</a>
             </li>
+            <li>
+              <a href="/contact_us">Contact</a>
```

Comment:
@slorek I tried, but it raises an exception, undefined local variable or method `contact_us_page_path'
I don't think that page (it is swagger documentation for the API) has visibility of the route helpers apart from other quirkiness judging by how it is written and the comments in there. It doesn't use our application layout, etc.